cfme/tests/automate/test_service_dialog.py

```diff
@@ -239,3 +239,33 @@ def test_radiobutton_dialog_element(appliance, request):
     }
     dialog, element = create_dialog(appliance, element_data)
     request.addfinalizer(dialog.delete_if_exists)
+
+
+def test_remove_dialog_element(appliance, request):
```

Comment:
please add a doc string to explain the test case and mention the BZ number that you are automating.